source/Octopus.Tentacle.Tests.Integration/Util/Builders/Decorators/TentacleServiceDecoratorBuilderExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static TentacleServiceDecoratorBuilder RecordMethodUsages(this TentacleSe
             var localMethodUsages = new MethodUsages();
             recordedUsages = localMethodUsages;
 
-            return builder.RegisterProxyDecorator(testCase.LatestScriptServiceType, service => MethodUsageProxyDecorator.Create(testCase.LatestScriptServiceType, service, localMethodUsages));
+            return builder.RegisterProxyDecorator(testCase.ScriptServiceToTest, service => MethodUsageProxyDecorator.Create(testCase.ScriptServiceToTest, service, localMethodUsages));
```

Comment:
nit: The `Type` of the script service is just needed in this class. I would love to see all usages of `Type` when referring to the script service be replaced with the enum and just mapped when used here